x-pack/plugins/security_solution/public/cloud_security_posture/components/vulnerabilities/vulnerabilities_preview.tsx

```diff
@@ -103,49 +95,22 @@ export const VulnerabilitiesPreview = ({
 
   const { euiTheme } = useEuiTheme();
 
-  const { data: dataMisconfiguration } = useMisconfigurationPreview({
-    query: buildEntityFlyoutPreviewQuery('host.name', name),
-    sort: [],
-    enabled: true,
-    pageSize: 1,
-  });
```

Comment:
it seems we didn't use `ignore_unavailable` here, but we'll do so now with the hook. How will that affect the behavior?

Reply:
good catch, I left this one when debugging an issue, will remove it